src/components/Dashboard/PendingFiling.vue

```diff
@@ -7,6 +7,14 @@
     <p>If this issue persists, please contact us.</p>
 
     <ErrorContact />
+    <div class="to-dashboard-container">
```

Comment:
Added a return to dashboard button for now, waiting for mockup updates

Reply:
Mock ups were updated to just add a `Return to Dashboard` button, which matches the implementation.